filip/clients/ngsi_ld/cb.py

```diff
@@ -297,557 +298,409 @@ def get_entity(self,
             self.log_error(err=err, msg=msg)
             raise
 
-    def get_entity_attributes(self,
-                              entity_id: str,
-                              entity_type: str = None,
-                              attrs: List[str] = None,
-                              response_format: Union[AttrsFormat, str] =
-                              AttrsFormat.NORMALIZED,
-                              **kwargs
-                              ) -> \
-            Dict[str, Union[ContextProperty, ContextRelationship]]:
-        """
-        This request is similar to retrieving the whole entity, however this
-        one omits the id and type fields. Just like the general request of
-        getting an entire entity, this operation must return only one entity
-        element. If more than one entity with the same ID is found (e.g.
-        entities with same ID but different type), an error message is
-        returned, with the HTTP status code set to 409 Conflict.
+    def get_entity_list(self,
+                        entity_id: Optional[str] = None,
+                        id_pattern: Optional[str] = ".*",
+                        entity_type: Optional[str] = None,
+                        attrs: Optional[List[str]] = None,
+                        q: Optional[str] = None,
+                        georel: Optional[str] = None,
+                        geometry: Optional[GeometryShape] = None,  # So machen oder wie auch für response_format
+                        coordinates: Optional[str] = None,
+                        geoproperty: Optional[str] = None,
+                        csf: Optional[str] = None,
+                        limit: Optional[PositiveInt] = 100,
+                        response_format: Optional[Union[AttrsFormat, str]] = AttrsFormat.NORMALIZED.value,
+                        ) -> Union[Dict[str, Any]]:
```

Comment:
@iripiri The type hint should be adjusted because normally, a list of ContextLDEntity will be returned.

Reply:
Done